src/main/java/sirius/kernel/async/AsyncExecutor.java

```diff
@@ -63,15 +63,14 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
         try {
             // If the executor is used by another Java framework, we cannot recover anything and
             // simply try to execute while blocking the caller...
-            if (!(r instanceof ExecutionBuilder.TaskWrapper)) {
+            if (!(r instanceof ExecutionBuilder.TaskWrapper wrapper)) {
                 r.run();
```

Comment:
Optional: call on wrapper not r

Reply:
no, there is a ! involved. but this makes the scoping of wrapper a little hard to read